tests/users/test_dao.py

```diff
@@ -148,6 +150,149 @@ def test_dao_delete_only_user_admin(self):
             (messages.USER_CANT_DELETE, HTTPStatus.BAD_REQUEST), dao_result
         )
 
+    def test_dao_delete_user(self):
+        dao = UserDAO()
+
+        user = UserModel(
+            name=user2["name"],
+            email=user2["email"],
+            username=user2["username"],
+            password=user2["password"],
+            terms_and_conditions_checked=user2["terms_and_conditions_checked"],
+        )
```

Comment:
We can use dictionary unpacking here
```suggestion
        user = UserModel(**user2)
```

Reply:
Also, as user2 is needed in other tests too, I will suggest moving it to a setUp function instead